### PR TITLE
Group B: Kim Coterwas 

### PR DESCRIPTION
**Pull request by KCoterwas**

### CODE REVIEW

File: _posts/2020-06-05-Kim.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: A place I'd like to visit when things get better...
+author: Kim Coterwas
+date: 2020-06-05
+---
+
+I'm looking forward to playing air hockey at Dave and Busters with my brother.
+
+![Air Hockey](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F75787206206008293%2F&psig=AOvVaw1-WEXAV0L3rP522aWp0fOZ&ust=1591452544607000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJClwO_s6ukCFQAAAAAdAAAAABAE)
```
Comment (zach004) [inline _posts/2020-06-05-Kim.md]: Image file is a non-valid link

Comment (KCoterwas) [reply _posts/2020-06-05-Kim.md]: probs cause it was from pinterest whoops

File: _posts/2020-06-05-Kim.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Comment your reply here.
```
Comment (John-Henares2) [inline _posts/2020-06-05-Kim.md]:
```suggestion
https://www.quotemaster.org/images/d2/d27bc480199b7f0f5414a268af8ed8d9.jpeg
```

File: _posts/2020-06-05-Kim.md (hunk 1 above)
Comment (KCoterwas) [reply _posts/2020-06-05-Kim.md]: [Air Hockey](https://media-cdn.tripadvisor.com/media/photo-s/06/6e/ef/c2/dave-and-buster-s.jpg)

Comment (KCoterwas) [inline _posts/2020-06-05-Kim.md]:
```suggestion
![Air Hockey](https://media-cdn.tripadvisor.com/media/photo-s/06/6e/ef/c2/dave-and-buster-s.jpg)
```

File: _posts/2020-06-05-Kim.md [hunk 3]
```diff
@@ -6,6 +6,6 @@ date: 2020-06-05
 
 I'm looking forward to playing air hockey at Dave and Busters with my brother.
 
-![Air Hockey](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F75787206206008293%2F&psig=AOvVaw1-WEXAV0L3rP522aWp0fOZ&ust=1591452544607000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCJClwO_s6ukCFQAAAAAdAAAAABAE)
+![Air Hockey](https://media-cdn.tripadvisor.com/media/photo-s/06/6e/ef/c2/dave-and-buster-s.jpg)
 
 https://www.quotemaster.org/images/d2/d27bc480199b7f0f5414a268af8ed8d9.jpeg
```
Comment (zach004) [inline _posts/2020-06-05-Kim.md]: Nice! Who usually wins?
-Zach

Comment (KCoterwas) [reply _posts/2020-06-05-Kim.md]: I'd say it's usually 6/10 times me, but don't ask my brother John ;)

File: _posts/2020-06-05-Kim.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+title: A place I'd like to visit when things get better...
+author: Kim Coterwas
+date: 2020-06-05
+---
+
+I'm looking forward to playing air hockey at Dave and Busters with my brother.
+
+![Air Hockey](https://media-cdn.tripadvisor.com/media/photo-s/06/6e/ef/c2/dave-and-buster-s.jpg)
+
+https://www.quotemaster.org/images/d2/d27bc480199b7f0f5414a268af8ed8d9.jpeg
```
Comment (asingh9) [inline _posts/2020-06-05-Kim.md]:
```suggestion
![pie chart](https://www.quotemaster.org/images/d2/d27bc480199b7f0f5414a268af8ed8d9.jpeg)
```

**Pull request merged**